File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        l = []
        cnt = 0
        for i in nums:
            if i == 0:
                l.append(cnt)
                cnt = 0
            else:
                cnt += 1
            l.append(cnt)
        return max(l)

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int cnt = 0;
        int max = 0;
        for (int i=0; i<nums.size(); i++){
            if (nums[i]==1){
                cnt += 1;
            }
            else{
                if (cnt>max){
                    max = cnt;
                }
                cnt = 0;
            }
        }
        if (cnt>max){max=cnt;}
        return max;
    }
};